### DISCUSSION TITLE
Wrong digest in docker 'minor' update PR

### DISCUSSION THREAD

**Discussion by snazy** [renovatebot/renovate#29366] [Request Help]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Renovatebot opened [this PR](https://github.com/projectnessie/nessie/pull/8675) today. It's a "minor" version update including a change of the image digest of the `mariadb` mulit-platform image from `mariadb:11.3.2@sha256:e101f9db31916a5d4d7d594dd0dd092fb23ab4f499f1d7a7425d1afd4162c4bc` to 11.4.2 - but the digest that renovatebot added is wrong, so the update-PR fails. Renovate added `eb37f58646a901dc7727cf448cae36daaefaba79de33b5058dab79aa4c04aefb` as the digest, but the correct one is `191810ff836b08017823d248a296702b6ea1d029572f92b15d606d3c2d2a90c5`. Previous bumps in that file were "just" digest updates ([here](https://github.com/projectnessie/nessie/commit/71f93ead257c6c3706e0c7d726ff906dd13c762d) and [here](https://github.com/projectnessie/nessie/commit/7acd26db5e553f9f8e5861e7f2a47db87cec31ce)).

I tried to figure out where the wrong digest `eb37f58646a901dc7727cf448cae36daaefaba79de33b5058dab79aa4c04aefb` comes from, but could not find it until Docker hub rate limited my browser :facepalm: .

We've just somewhat recently updated our renovate-config to use/force digests. The current config is [here](https://github.com/projectnessie/nessie/blob/ca6d2e85483d0de5c1c2be456170562cb5703c90/.github/renovate.json5).

We've also seen a bunch of working updates for other images. Not sure whether it's 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by snazy** [comment]
For reference (in case the PR gets updated), [this is the original PR commit](https://github.com/projectnessie/nessie/pull/8675/commits/de63de9485c9e3c40987c079d901f53936a2c2c7)

**Comment by snazy** [comment]
Hm - yea - temporary issue. Renovatebot [updated the PR](https://github.com/projectnessie/nessie/commit/bfda0a469997171714e6a0d1c434630bbf44ecf4) with the correct digest. So no issue.

**Reply by rarkins** [reply]
When images are published they are usually done one platform at a time so hashes can take a while to stabilize
